data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+			`	There is a pause. "I see that you can not be convinced by me. Even though I could simply wipe you out here and now, I do not favor violence." The voice is now resolute in tone. "I will let you pass, but I will be watching you, as I am the guardian of this space."`
+			`	The voice fades out leaving nothing behind but the feeling of being watched.`
+				decline
+			label agree
+			action
+				set "Incipias: accepted the guardians warning"
+				event "label incipias space"
+			`	"You are understanding, human, so I thank you in the name of the Incipias." The voice fades away, leaving only the feeling of being watched."`
+				decline
+
+mission "Incipias: Presence of the Guardian"
+	invisible
+	landing
+	to complete
+		never
+	on enter Dom'us
+		fail
+		conversation
+			`As you enter the system you have a feeling of being watched. The presence from the previous system still lingers.`
+				to display
+					not "Incipias: accepted the guardians warning"
+			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`
+				to display
+					has "Incipias: accepted the guardians warning"
+				decline
+	on enter Hui'uc
+		fail
+		conversation
+			`As you enter the system you have a feeling of being watched. The presence from the previous system still lingers.`
+				to display
+					not "Incipias: accepted the guardians warning"
+			`As you enter the system the same voice from before echoes through your mind. "I do not take kindly on attempts to deceive me, human. Know that you will be watched." The voice fades away, but the strong presence remains.`
+				to display
+					has "Incipias: accepted the guardians warning"
+				decline
+
+
+event "label incipias space"
+	galaxy "label incipias"
+		sprite "label/incipias"
+
+mission "Incipias: First Contact"
+	invisible
+	landing
+	to complete
+		never
+	on enter Dom'us
+		conversation
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
+				to display
+					not "Visit Quarg in Incipias Space: offered"
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
+				to display
+					has "Visit Quarg in Incipias Space: offered"
```

Comment:
I'm not sure these really need to describe the player hailing the ships. They would do so curiously outside of telling them; I think it's better to leave this action up to the player performing it themselves rather than tell them they did it before they even try to in-game.

If the purpose is to lead into the line to explore other nearby systems, I also don't think that's strictly necessary - this is a brand new area, the player will explore eventually find the Quarg anyways.

Though I'm not sure of a good alternative at the moment.

Reply:
Hm how about they hail you?

Reply:
"speakers stay silent but a colorful haze appears on your screen." smth like that

Reply:
I feel saying they hail you here would take away from the "first direct hail" you get with the scene. It could just be that they don't try to hail you initially, and that's what you say. Perhaps they're hesitant to start with.